util/math_reference.cpp

```diff
@@ -336,6 +345,7 @@ T mul_hi_signed(T a, T b) {
   r = static_cast<U>(a_pos) & mask;
   s = static_cast<U>(b_pos) & mask;
 
+  // calc half products
   lo = r * s;
   hi = p * q;
   cross1 = (p * s);
```

Comment:
You can remove the parenthesis while you are here. :-)

Reply:
Why not using `std::abs` above, replacing `typedef` by `using`, make `a_pos` and `b_pos` just a `U` and remove the `static_cast` later...

Reply:
Fixed it. It certainly looks better. Thanks!